app/views/badges/_badge.html.erb

```diff
@@ -1,6 +1,13 @@
 <div class="row">
-    <div class="col-10">
-        <%= link_to(image_tag(badge.image.url(:thumb)),badge_path(badge)) %>
+    <div class="col-4">
+        <%= link_to(image_tag(badge.image.url(:thumb), alt: badge.title),badge_path(badge)) %>
         <%= badge.title %>
     </div>
+    <div class="col-5 my-auto">
+        <% if badge.users.where(id: current_user).exists? %>
```

Comment:
@TheDudeWithTheThing  I need to provide a different link here for create or destroy depending on if the badge is attached to the `current_user` or not.  This is an N+1 query  as it wants to hit the database for each badge as it's displayed to see if it's attached.  

Any ideas on how to do this better?